src/Breadcrumb.php

```diff
@@ -357,6 +362,22 @@ public function toArray(): array
         ];
     }
 
+    /**
+     * Constructon helper.
+     *
+     * @param array $array The set of construction key/value pairs
+     */
+    public static function fromArray(array $array): self
+    {
+        return new self(
+            $array['level'],                      // level is required
+            $array['type'] ?? self::TYPE_DEFAULT, // use default if not supplied
+            $array['category'],                   // category is required
+            $array['message'] ?? (string) null,   // use null cast to string if no value supplied
+            $array['data'] ?? []                  // supply empty array if none was supplied
```

Comment:
Please remove such comments since it provides no real value (it's obvious from the usage of the `??` statement that we are doing fallback to a default)